src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/controllers/web/ReportControllerTests/RunReportTests.kt

```diff
@@ -119,9 +119,10 @@ class RunReportTests
     @Test
```

Comment:
import org.junit.Test issue here too.
- package name should start with lowerCase `reportControllerTests` rather than `ReportControllerTests`.

Reply:
This isn't related to any changes made in this PR, so I think strictly this needn't be a requested change - but will do so opportunistically!